salt/modules/mac_system.py

```diff
@@ -66,6 +67,87 @@ def _execute_command(cmd, at_time=None):
     return not bool(__salt__['cmd.retcode'](cmd, python_shell=True))
 
 
+def _execute_return_success(cmd):
+    '''
+    Helper function to execute the command
+    Returns: bool
+    '''
+    ret = __salt__['cmd.run_all'](cmd)
+
+    if 'not supported' in ret['stdout'].lower():
+        return 'Not supported on this machine'
+
+    if ret['retcode'] != 0:
+        msg = 'Command Failed: {0}\n'.format(cmd)
+        msg += 'Return Code: {0}\n'.format(ret['retcode'])
+        msg += 'Output: {0}\n'.format(ret['stdout'])
+        raise CommandExecutionError(msg)
+
+    return True
+
+
+def _execute_return_result(cmd):
+    '''
+    Helper function to execute the command
+    Returns: the results of the command
+    '''
+    ret = __salt__['cmd.run_all'](cmd)
+
+    if ret['retcode'] != 0:
+        msg = 'Command failed: {0}'.format(ret['stderr'])
+        raise CommandExecutionError(msg)
+
+    return ret['stdout']
+
+
+def _parse_return(data):
+    '''
+    Parse a return in the format:
+    ``Time Zone: America/Denver``
+    to return only:
+    ``America/Denver``
+
+    Returns: The value portion of a return
+    '''
+
+    if ': ' in data:
+        return data.split(': ')[1]
+    if ':\n' in data:
+        return data.split(':\n')[1]
+    else:
+        return data
+
+
+def _validate_enabled(enabled):
+    '''
+    Helper function to validate the enabled parameter. Boolean values are
+    converted to "on" and "off". String values are checked to make sure they are
+    either "on" or "off". All other values return an error.
+
+    Returns: "on" or "off" or errors
+    '''
+    if isinstance(enabled, bool):
+        if enabled:
+            return 'on'
+        else:
+            return 'off'
+    elif isinstance(enabled, int):
+        if enabled in [1, 0]:
+            if enabled == 1:
+                return 'on'
+            else:
+                return 'off'
```

Comment:
It does, under normal circumstances. But if someone knew "on" or "off" were desired they might double quote to make sure yaml didn't convert. Anyway, now it matches the docstring, which is what I really wanted.